### PR TITLE
update readme

### LINKED ISSUE #40: update readme install instructions
@jakirkham, i suspect i can update/simplify the current readme suggestions.  I think specifying cudatoolkit might be outdated?
and, do you agree i should just link to https://docs.nvidia.com/deploy/cuda-compatibility/ for the compatibility table?

-----

### GPU requirements

This software requires a CUDA-compatible NVIDIA GPU.
The libraries available on conda-forge have been compiled against different versions of the CUDA toolkit.  The required CUDA libraries are bundled in the conda distributions so you don't need to install the CUDA toolkit separately.  If desired, you can pick which version of CUDA you'd like based on your needs, but please note that different versions of the CUDA toolkit have different GPU driver requirements:

To specify a specific cudatoolkit version, install as follows (for instance, to use
`cudatoolkit=10.2`)

```sh
conda install -c conda-forge cudadecon cudatoolkit=10.2
```

| CUDA  | Linux driver | Win driver |
| ----- | ------------ | ---------- |
| 10.2  | ≥ 440.33     | ≥ 441.22   |
| 11.0  | ≥ 450.36.06  | ≥ 451.22   |
| 11.1  | ≥ 455.23     | ≥ 456.38   |
| 11.2  | ≥ 460.27.03  | ≥ 460.82   |

### PR DESCRIPTION
**Pull request by tlambert03**
closes #40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,40 +74,44 @@ configs available in `.ci_support`, for instance:
...
+   VisualStudio](https://visualstudio.microsoft.com/downloads)
+   2019.  For linux, all necessary build tools will be installed by conda.
 
-4. create a new conda environment with all of the dependencies installed
+1. create a new conda environment with all of the dependencies installed
 
     ```sh
     conda config --add channels conda-forge
-    conda create -n build -y cmake boost-cpp libtiff fftw ninja
+    conda create -n build -y cmake libboost-devel libtiff fftw ninja cuda-nvcc libcufft-dev
```
Comment (jakirkham) [inline README.md:95]: Note this only works for CUDA 12. That said, this is a reasonable choice from a docs perspective

**Review by jakirkham — COMMENTED** [review]
Thanks Talley! 🙏

Had a minor comment below. No need to change. Just wanted to clarify something

File: README.md (hunk 1 above)
Comment (tlambert03) [reply README.md:95]: yeah, thank you.  i definitely used those locally and missed it here

Comment (tlambert03) [reply README.md:95]: oh, sorry :joy: yes, you're pointing out that it's cuda12 specific.  Yeah, i could include that ...

Comment (jakirkham) [reply README.md:95]: Yeah

Though starting developers off with the newer thing seems more reasonable and should be easier to get started with

**Pull request merged**